### DISCUSSION TITLE
Does ArgoCD support private helm repo hosted in GCP?

### DISCUSSION THREAD

**Discussion by FL3SH** [argoproj/argo-cd#5070] [Q&A] [answered]
Does ArgoCD support private helm repo hosted in GCP?

**Comment by jessesuen** [comment, marked as answer]
It's intended to work, but there's a bug preventing this from working properly as of v1.8.1. You can follow https://github.com/argoproj/argo-cd/issues/5030.

**Reply by FL3SH** [reply]
thank you
